spk/flexget/src/requirements.txt

```diff
@@ -1,33 +1,36 @@
-paver
-FeedParser>=5.1.3
-argparse
-babelfish>=0.5.4
-stevedore>=0.14
-BeautifulSoup
-beautifulsoup4>=4.1, !=4.2.0, <4.4
-plumbum
-rpyc
-path.py
+FeedParser>=5.2.1
```

Comment:
See the WIKI article on wheels: https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-packages. 
All wheels must be pinned to a specific version, and all requirements should be listed here (unless they are providedvia the Python package, e.g. setuptools, wheel, pip, other included cross-compiled modules, and modules listed in Pythones requirements.txt).

For Flexget, the current complete list looks like this:
```
aniso8601==1.2.0
APScheduler==3.3.1
babelfish==0.5.5
beautifulsoup4==4.5.3
cheroot==5.1.0
CherryPy==10.0.0
click==6.7
colorclass==2.2.0
feedparser==5.2.1
Flask==0.12
Flask-Compress==1.4.0
Flask-Cors==3.0.2
Flask-Login==0.4.0
Flask-RESTful==0.3.5
flask-restplus==0.8.6
FlexGet==2.9.11
funcsigs==1.0.2
functools32==3.2.3.post2
future==0.16.0
futures==3.0.5
guessit==2.0.4
html5lib==0.999999999
itsdangerous==0.24
Jinja2==2.9.5
jsonschema==2.5.1
MarkupSafe==0.23
path.py==10.1
pathlib==1.0.1
plumbum==1.6.3
portend==1.8
pynzb==0.1.0
pyparsing==2.1.10
PyRSS2Gen==1.1
python-dateutil==2.6.0
pytz==2016.10
PyYAML==3.12
rebulk==0.8.2
requests==2.13.0
rpyc==3.3.0
six==1.10.0
SQLAlchemy==1.1.5
tempora==1.6.1
terminaltables==3.1.0
tzlocal==1.3
webencodings==0.5
Werkzeug==0.11.15
wheel==0.24.0
zxcvbn-python==4.4.14
```
As mentioned above, note that there may be modules that need cross-compiling in this list, as well as default modules provided by the Python package. Those may need to be excluded/commented out (e.g. PyYAML is one of those).